lib/pbench/cli/agent/commands/tools/clear.py

```diff
@@ -122,6 +55,81 @@ def execute(self) -> int:
 
         return errors
 
+    def _clear_remotes(self, group) -> Union[int, list]:
```

Comment:
That looks like a `Tuple`, not a `Union`! Also, I think it's `Tuple[int,List[str]]`.

And I think we need a docstring here: particularly with an anonymous tuple return. In fact ... type hint on `group`, too!